critiquebrainz/frontend/templates/artist/entity.html

```diff
@@ -48,7 +52,7 @@ <h2 class="pull-left">
   </h2>
 
   {% if not my_review %}
-    <a id="write-review" href="{{ url_for('review.create', artist=id) }}"
+    <a id="write-review" href="{{ url_for('review.create', entity_type='artist', entity_id=id) }}"
```

Comment:
this change seems unrelated to this bug (is it from #298?). Please make sure that you don't include unrelated changes in commits.

Reply:
Yeah right. I'll take care of it from now on.